### DISCUSSION TITLE
Re-routing

### DISCUSSION THREAD

**Discussion by davidwilcoxturbulence** [TUDelft-CNS-ATM/bluesky#396] [Q&A]
Hi,
Are there any re-routing algorithms in BlueSky? Say for example, I have a restricted zone and I want to go around this area, would BlueSky offer something like this?

**Comment by jooste** [comment]
Hi @davidwilcoxturbulence,

This is currently not a core part of BlueSky, but there are several implementations of this in different forks of BlueSky, see for instance the Metropolis 2 fork: https://github.com/Metropolis-2/bluesky.
